pipsi.py

```diff
@@ -209,6 +210,34 @@ def link_scripts(self, scripts):
 
         return rv
 
+    def get_package_version(self, venv_path, package):
+        from subprocess import Popen, PIPE
+
+        args = [os.path.join(venv_path, BIN_DIR, 'pip'), 'freeze']
```

Comment:
Why? It's used in the same way as in install:

`args = [os.path.join(venv_path, BIN_DIR, 'pip'), 'install']`

I have no easy way to test it on windows, but it should work fine.